clients/drcachesim/tracer/tracer.cpp

```diff
@@ -2662,7 +2671,7 @@ event_thread_init(void *drcontext)
 {
     per_thread_t *data = (per_thread_t *)dr_thread_alloc(drcontext, sizeof(per_thread_t));
     DR_ASSERT(data != NULL);
-    memset(data, 0, sizeof(*data));
+    *data = {};
```

Comment:
Why was this change needed?

Reply:
The compiler warns about zeroing a no-longer-POD struct: this is how you're supposed to zero w/o zeroing the class (and potentially clobbering vtable).

Reply:
Added a comment.